src/plugins/arkeo/StakingCards.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+  cosmosAssetId,
+  cosmosChainId,
+  fromAssetId,
+  osmosisAssetId,
+  osmosisChainId,
+} from '@shapeshiftoss/caip'
+import { DefiProvider } from 'features/defi/contexts/DefiManagerProvider/DefiCommon'
+import type { EarnOpportunityType } from 'features/defi/helpers/normalizeOpportunity'
+import { useTransformCosmosStaking } from 'features/defi/helpers/normalizeOpportunity'
+import qs from 'qs'
+import { useCallback, useMemo } from 'react'
+import { useHistory, useLocation } from 'react-router'
+import { WalletActions } from 'context/WalletProvider/actions'
+import { useWallet } from 'hooks/useWallet/useWallet'
+import { useCosmosSdkStakingBalances } from 'pages/Defi/hooks/useCosmosSdkStakingBalances'
+import type { OpportunityId } from 'state/slices/opportunitiesSlice/types'
+import {
+  selectAggregatedEarnUserStakingOpportunitiesIncludeEmpty,
+  selectFirstAccountIdByChainId,
+} from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import { StakingCard } from './StakingCard'
+
+type StakingCardsProps = {
+  ids: OpportunityId[]
+}
+export const StakingCards: React.FC<StakingCardsProps> = ({ ids }) => {
+  const history = useHistory()
+  const location = useLocation()
+  const {
+    state: { isConnected, isDemoWallet },
+    dispatch,
+  } = useWallet()
+  const stakingOpportunities = useAppSelector(
+    selectAggregatedEarnUserStakingOpportunitiesIncludeEmpty,
+  )
+  const cosmosAccountId = useAppSelector(state =>
+    selectFirstAccountIdByChainId(state, cosmosChainId),
+  )
+  const osmosisAccountId = useAppSelector(state =>
+    selectFirstAccountIdByChainId(state, osmosisChainId),
```

Comment:
Is Arkeo only intended to be working with account 1 for the time being? Otherwise we'll be able to select the highest balance ID soon

Reply:
Disregard me, this is related to the highly cosmos-specific  `defaultAccountId` logic we should rename, and make sure not to remove from the routing for assets other than Cosmos and Osmosis